ui/src/components/dialog/__tests__/DialogWrapper.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div data-cy="dialog-page">
+    <q-dialog ref="dialogRef" v-bind="$attrs" data-cy="dialog-form">
```

Comment:
`dialog-form` doesn't seem a good name for this
`dialog-container` maybe? Or just `dialog`? Or whatever

If you can avoid referencing this altogether in tests, that would be even better